devices/lenovo-krane/default.nix

```diff
@@ -0,0 +1,63 @@
+{ config, lib, pkgs, ... }:
+
+{
+  imports = [
+    ./sound.nix
+  ];
+
+  mobile.device.name = "lenovo-krane";
+  mobile.device.identity = {
+    name = "Chromebook Duet";
+    manufacturer = "Lenovo";
+  };
+  mobile.device.supportLevel = "supported";
+
+  mobile.hardware = {
+    soc = "mediatek-mt8183";
+    ram = 1024 * 4;
+    screen = {
+      # Panel is portrait CW compared to keyboard attachment.
+      width = 1200; height = 1920;
+    };
+  };
+
+  mobile.boot.stage-1 = {
+    kernel.package = pkgs.callPackage ./kernel {};
+    kernel.modular = true;
+    kernel.additionalModules = [
+      # Breaks udev if builtin or loaded before udev runs.
+      # Using `additionalModules` means udev will load them as needed.
+      "sbs-battery"
+      "sbs-charger"
```

Comment:
With an eventual rework of `asus-dumo`, this might be extracted into a shared `chrome-sbs` quirk, which would force `=m` to the kernel config, and add those to `addtionalModules`.